src/Extractor/SwfExtractor.php

```diff
@@ -252,10 +253,13 @@ public function exported(): array
 
         $exported = [];
 
-        foreach ($this->file->tags(ExportAssetsTag::ID) as $tag) {
-            assert($tag instanceof ExportAssetsTag);
-
-            $exported += array_flip($tag->characters);
+        foreach ($this->file->tags(ExportAssetsTag::ID, SymbolClassTag::TYPE) as $tag) {
+            if ($tag instanceof ExportAssetsTag) {
+                $exported += array_flip($tag->characters);
+            }
+            elseif ($tag instanceof SymbolClassTag) {
```

Comment:
```suggestion
            } elseif ($tag instanceof SymbolClassTag) {
```

Il n'y a pas de nouvelle ligne entre le } et elseif